src/main/scala/com/advancedtelematic/campaigner/data/DataType.scala

```diff
@@ -90,13 +93,15 @@ object DataType {
     status: CampaignStatus,
     createdAt: Instant,
     updatedAt: Instant,
+    parentCampaignId: Option[CampaignId],
+    childCampaignIds: Set[CampaignId],
     groups: Set[GroupId],
     metadata: Seq[CreateCampaignMetadata],
     autoAccept: Boolean
   )
 
   object GetCampaign {
-    def apply(c: Campaign, groups: Set[GroupId], metadata: Seq[CampaignMetadata]): GetCampaign =
```

Comment:
I was thinking, do we actually want to send the child campaigns? I don't think we'll need them. Only the child campaign stats when requesting the stats of the parent.

Reply:
Unfortunately I don't have enough knowledge to address this, but I think we might need them to actually get the stats for them. WDYT @jerrytrieu?